spec/functional/resource/user/useradd_spec.rb

```diff
@@ -77,9 +76,19 @@ def password_should_be_set
     end
   end
 
+  def try_cleanup
+    ['/home/foo', '/home/bar'].each do |f|
+      shell_out("rm", "-r", f)
```

Comment:
I'm a little nervous about ```rm -rf /home/foo /home/bar```. While "foo" and "bar" are common through away names, if someone runs ```rake spec``` on their workstation, they're not going to expect this. Maybe they've got some stuff they don't want to throw away there. We should change the names in this test to something that's not so common.